tests/issues/test_188_concurrency.py

```diff
@@ -3,14 +3,15 @@
 from pydantic import AnyUrl
 
 from mcp.server.fastmcp import FastMCP
-from mcp.shared.memory import (
-    create_connected_server_and_client_session as create_session,
-)
+from mcp.shared.memory import create_connected_server_and_client_session as create_session
 
 _sleep_time_seconds = 0.01
 _resource_name = "slow://slow_resource"
 
 
+@pytest.mark.filterwarnings(
+    "ignore:coroutine 'test_messages_are_executed_concurrently.<locals>.slow_resource' was never awaited:RuntimeWarning"
+)
```

Comment:
I couldn't find the reason or package that triggered this - I tried bumping anyio, but that didn't seem to be the reason - but this seems an issue only with the test suite, so it doesn't have any impact.